packages/g/gdu/monitoring.yaml

```diff
@@ -0,0 +1,5 @@
+releases:
+  id:
```

Comment:
This project is on the [Release Monitoring webpage](https://release-monitoring.org/project/141586/), so the `id` here should be 141586.